src/main/java/dataProviders/ConfigFileReader.java

```diff
@@ -58,7 +58,7 @@ public long getImplicitlyWait() {
 	public String getApplicationUrl() {
 		String url = properties.getProperty("applicationURL");
 		if(url != null) {
-			return "https://" + url;
+			return "https://www." + url;
```

Comment:
If URL will contains string www then it can be directly added into the config properties

Reply:
> If URL will contains string www then it can be directly added into the config properties

Yes. URL contains www. Ok will add in Config file.

Reply:
Done. Please review